gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/validation/ValidationJob.java

```diff
@@ -301,7 +340,8 @@ private void processPartitionedTable(ConvertibleHiveDataset hiveDataset, AutoRet
     Closer closer = Closer.create();
 
     try {
-      statement = this.hiveJdbcConnector.getConnection().createStatement();
+      HiveJdbcConnector hiveJdbcConnector = HiveJdbcConnector.newConnectorWithProps(props);
```

Comment:
The maximum number of connections is equal to the number of threads (Default 50).
I fixed the code to close.
The dbcp change should be made in ```gobblin.hive.util.HiveJdbcConnector``` I will make that in a different PR.